add CONTRIBUTING.md and update README (#11)
expo/turtle: CONTRIBUTING.md

# Contributing to Turtle CLI

## How to contribute
You can contribute to Turtle CLI in various ways, including:

> - [Reporting bugs or issues](https://github.com/expo/turtle/issues/new) on GitHub. Please make sure to include fill in all the details in the issue template to make sure the issue can be addressed as quickly as possible.

Reviewer: ```suggestion
- [Reporting bugs or issues](https://github.com/expo/turtle/issues/new) on GitHub. Please make sure to fill in all the details suggested by the template to make sure the issue can be addressed as quickly as possible.
```

> - [Submitting improvements to the documentation](https://github.com/expo/expo/tree/master/docs). Updates, enhancements, new guides, spelling fixes...

Reviewer: ```suggestion
- [Submitting improvements to the documentation](https://github.com/expo/expo/tree/master/docs). Updates, enhancements, new guides, spelling fixes…
```

> - [Submitting improvements to the documentation](https://github.com/expo/expo/tree/master/docs). Updates, enhancements, new guides, spelling fixes...

Reviewer: ```suggestion
- [Submitting improvements to the documentation](https://github.com/expo/expo/tree/master/docs). Updates, enhancements, new guides, spelling fixes…

```
